builder/azure/arm/builder_acc_test.go

```diff
@@ -10,6 +10,8 @@ package arm
 // * ARM_CLIENT_SECRET
 // * ARM_SUBSCRIPTION_ID
 // * ARM_STORAGE_ACCOUNT
+// * AZURE_SSH_PASS - In our Linux SIG test we build two specialized images, this requires setting a shared password for both builds
+// * ARM_SSH_PRIVATE_KEY_LOCATION - the file location of a PEM encoded SSH private key,
```

Comment:
Is there a reason we want to use an external key and not one we generate on the fly?

Reply:
Only that I thought it would be simpler, certainly open to generating this on the fly, my plan was to just create and delete this key in CI when we start running acceptance tests there, I'll try to make this change!

Reply:
I don't have an suggestions at this point but the requirements for env var in this test is a bit involved. Maybe AZURE_CLI_AUTH not being set should also be a Fatalf as it seems to be need for the new tests.

Reply:
I agree with that, we should just require the environment variables, skipping them makes it easier to miss that tests aren't running

Reply:
Added that requirement to relevant tests